src/xpk/commands/cluster.py

```diff
@@ -883,6 +884,10 @@ def run_gke_cluster_create_command(
   if args.enable_pd_csi_driver:
     addons.append('GcePersistentDiskCsiDriver')
 
+  if args.enable_lustre_csi_driver:
+    addons.append('LustreCsiDriver')
+    command += ' --enable-legacy-lustre-port'
```

Comment:
@sharabiani thanks for the great work on the Lustre and XPK integration!
Quick heads-up: the `enable-legacy-lustre-port` flag is now deprecated in the GKE API. It won't break anything for existing customers, but we need to ensure new users avoid it, which will require some modifications in XPK.

[This doc](https://cloud.google.com/managed-lustre/docs/lustre-csi-driver-new-volume#manage) has all the details. But feel free to ping me if you have any questions!